### DISCUSSION TITLE
Unsupported platform: linux (arm) - Raspbian

### DISCUSSION THREAD

**Discussion by Jierr** [streamlink/streamlink-twitch-gui#787] [Q&A] [answered]
Building (yarn install) on raspbian for raspberry 3b+ succeeds.

Running w/ **yarn run grunt release** fails.

```
yarn run v1.22.10

Running "release" task
Warning: Unsupported platform: linux (arm)
```

Is there any way to resolve this issue?

**Comment by bastimeyer** [comment, marked as answer]
This is a platform limitation. NW.js currently does only provide binaries for x86 and x86_64:
https://nwjs.io/downloads/

https://github.com/streamlink/streamlink-twitch-gui/blob/96d0d53b7bca1a6d40bad02aaf7b032fea3f31fa/build/tasks/common/platforms.js#L2-L25

You can try out downloading community builds for armv7, but there are no build configs for this here and you'll have to figure it out yourself.
